repos/system_upgrade/el7toel8/actors/persistentnetnames/actor.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pyudev
+
+from leapp.libraries.common import *
+from leapp.actors import Actor
+from leapp.models import PCIAddress, Interface, PersistentNetNamesFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+class PersistentNetNames(Actor):
+    name = 'persistentnetnames'
+    description = 'Gather current network naming configuration'
```

Comment:
We are not using description anymore. Document actors with docstrings: https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md#code-style  (see the example).